addons/im_livechat/__manifest__.py

```diff
@@ -73,6 +73,7 @@
             'im_livechat/static/src/core/common/**/*',
             'im_livechat/static/src/core/public_web/**/*',
             'im_livechat/static/src/core/web/**/*',
+            'im_livechat/static/src/embed/common/expirable_storage.js',
```

Comment:
Note: this is needed because it's used by `chatbot_model.js`

Reply:
The question is why is it needed in master and not 18.1?